tedana/tests/test_decay.py

```diff
@@ -32,15 +32,13 @@ def test_fit_decay(testdata1):
     """
     fit_decay should return data in (samples,) shape.
     """
-    t2sv, s0v, t2ss, s0vs, t2svG, s0vG = me.fit_decay(testdata1['data'],
-                                                      testdata1['tes'],
-                                                      testdata1['mask'],
-                                                      testdata1['mask_sum'],
-                                                      testdata1['fittype'])
+    t2sv, s0v, t2svG, s0vG = me.fit_decay(testdata1['data'],
+                                          testdata1['tes'],
+                                          testdata1['mask'],
+                                          testdata1['mask_sum'],
```

Comment:
Should all mentions of mask_sum become adaptive mask?

Reply:
As part of #448, absolutely, although I'd prefer to just do that in one fell swoop.

Reply:
Actually never mind. Might as well do it now.